### PR TITLE
updated yarn command

### LINKED ISSUE #24811: Yarn3 publish command
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

content\actions\publishing-packages\publishing-nodejs-packages.md

### What part(s) of the article would you like to see updated?

// For Yarn3 use -
// yarn npm publish
- run: yarn publish

lines 180 - 183

### Additional information

I should fork the repo only for such a small suggestion instead of branch push and PR ?
I believe the comment fits, your choice.
Have a great day =).

### PR DESCRIPTION
**Pull request by PaoloJr**
### Why:
request was made to update the yarn run command.

Closes:  #24811

### What's being changed (if available, include any code snippets, screenshots, or gifs):
line 179 was modified; it now indicates `- run: yarn npm publish // for Yarn version 1, use yarn publish instead` instead of `- run: yarn publish`

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@PaoloJr90 Thanks for submitting this PR! I've edited your post slightly to link it to the `help wanted` issue. 

I'll get this triaged for review! :zap:

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Secret gists don't show up in {% data variables.gists.discover_url %} and are no
 
 If your site administrator has disabled private mode, you can also use anonymous gists, which can be public or secret.
 
-{% data reusables.gist.anonymous-gists-cannot-be-deleted %}
+{% data reusables.gist.anonymous-gists-cannot-be-deleted %}.
```
Comment (subatoi) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:32]:
```suggestion
{% data reusables.gist.anonymous-gists-cannot-be-deleted %}
```

This doesn't need a period at the end as it's referencing content that already has one:

https://github.com/github/docs/blob/main/data/reusables/gist/anonymous-gists-cannot-be-deleted.md

**Review by subatoi — CHANGES_REQUESTED** [review]
Thank you for raising this PR! Just a tiny change needed and then we can get the other change live, that closes the original issue as previously discussed

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md (hunk 1 above)
Comment (PaoloJr90) [reply content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:32]: sounds good. I think Prettier or Markdownlint extensions have made those additions without me noticing - will be more vigilant on that 👍

**Review by subatoi — APPROVED** [review]
Thank you!

**Comment by subatoi** [comment]
Thanks for your interest and help in improving our docs, @PaoloJr90!

**Pull request merged**